Jenkinsfile

```diff
@@ -3,8 +3,10 @@ pipeline {
       label 'maven-21'
    }
 
-   triggers {
-      cron('H H * * *')
+   if (env.BRANCH_IS_PRIMARY) {
+      properties([pipelineTriggers([cron('H H * * *')])])
+   else {
+      properties([pipelineTriggers([cron('')])])
```

Comment:
or just `properties([])`

Reply:
I need to retry, but last time @smerle33 and I did try on infra.ci, it only worked when specifying an empty string to cron trigger.